gdp/column/column.py

```diff
@@ -543,181 +741,219 @@ def YP_or_notYP(m, n):
 
     @m.total_cond.Constraint(m.comps)
     def liquid_fraction_pass_through(_, c):
+        """This function ensures the mole fractions of the liquid phase are equal to those of the vapor phase from the tray below"""
         return m.x[c, t] == m.y[c, t - 1]
 
     @m.Constraint(m.comps)
     def condenser_distillate_composition(_, c):
+        """This function defines the composition of the distillate in the condenser tray"""
         return m.D[c] == m.dis * m.x[c, t]
 
 
 def _build_reboiler_mass_balance(m):
+    """This function defines the mass balance for each component in the reboiler tray"""
     t = m.reboil_tray
 
     @m.Constraint(m.comps)
     def reboiler_mass_balance(_, c):
+        """This function defines the mass balance for each component in the reboiler tray"""
         t = m.reboil_tray
+        """The equation considers the mass balance in terms of molar flow rates"""
         return (
-            - m.V[c, t]      # Vapor from tray t
-            + m.L[c, t + 1]  # Liquid from tray above
+            - m.V[c, t]      # Vapor leaving the tray
+            + m.L[c, t + 1]  # Liquid coming from the tray above
             - m.B[c]         # Loss to bottoms
             == 0)
 
     @m.Constraint(m.comps)
     def reboiler_liquid_composition(_, c):
+        """This function defines the composition of the liquid phase in the reboiler tray"""
         return m.L[c, t] == m.liq[t] * m.x[c, t]
 
     @m.Constraint(m.comps)
     def reboiler_vapor_composition(_, c):
+        """This function defines the composition of the vapor phase in the reboiler tray"""
         return m.V[c, t] == m.vap[t] * m.y[c, t]
 
-
 def _build_tray_phase_equilibrium(m, t, tray):
+    # This function defines Raoult's law for each component in a tray
     @tray.Constraint(m.comps)
     def raoults_law(_, c):
+        """Raoult's law for each component in a tray."""
         return m.y[c, t] == m.x[c, t] * m.Kc[c, t]
 
+    # This function defines the phase equilibrium constant for each component in a tray
     @tray.Constraint(m.comps)
     def phase_equil_const(_, c):
+        # The equation relates the phase equilibrium constant to the activity coefficient and the vapor pressure
         return m.Kc[c, t] * m.P == (
             m.gamma[c, t] * m.Pvap[c, t])
 
+    # This function defines the relationship between vapor pressure and temperature for each component in a tray
     @tray.Constraint(m.comps)
     def Pvap_relation(_, c):
         k = m.pvap_const[c]
         x = m.Pvap_X[c, t]
+        # The equation uses the Antoine equation to relate the vapor pressure 
+        # to the composition (represented by the variable x)
         return (log(m.Pvap[c, t]) - log(k['Pc'])) * (1 - x) == (
             k['A'] * x +
             k['B'] * x ** 1.5 +
             k['C'] * x ** 3 +
             k['D'] * x ** 6)
 
+    # This function defines the relationship between the temperature variable (x) and the temperature in a tray
```

Comment:
This comment is also wrong. 'x' is not the Temperature and it is not actually used in the constraints below

Reply:
Clarified the Pvap_X. Make the comment more clear.